subschemas/substrate-api-proxy/src/resolvers/Query/council.ts

```diff
@@ -42,26 +42,33 @@ export async function council(
   const members = electionsInfo.members.map<PartialCouncilMember>(([accountId, balance]) => ({
     address: String(accountId),
     backing: balance.toString(),
+    formattedBacking: formatBalance(api, balance),
     voters: votesByCandidates[String(accountId)] || [],
   }));
 
   const runnersUp = electionsInfo.runnersUp.map<PartialCouncilMember>(([accountId, balance]) => ({
     address: String(accountId),
     backing: balance.toString(),
+    formattedBacking: formatBalance(api, balance),
     voters: votesByCandidates[String(accountId)] || [],
   }));
 
   const candidates = electionsInfo.candidates.map<PartialCouncilCandidate>((accountId) => ({
     address: String(accountId),
   }));
 
-  const primeMember: PartialCouncilMember | null = prime
-    ? {
-        address: String(prime),
-        backing: electionsInfo.members.find(([accountId]) => accountId.eq(prime))?.[1]?.toString(),
+  let primeMember: PartialCouncilMember | null = null;
+  if (prime) {
+    const backing = electionsInfo.members.find(([accountId]) => accountId.eq(prime))?.[1];
+    if (backing) {
+      primeMember = {
+        address: prime.toString(),
+        backing: backing?.toString() as string,
+        formattedBacking: formatBalance(api, backing),
         voters: [],
```

Comment:
@franjs Prime members don't have voters?

Reply:
They do, we just use this to display who the prime voter is in the council summary. In the council list we get the voters